core/src/worker/data-feed.ts

```diff
@@ -184,6 +185,12 @@ export function aggregatorJob(
     logger.debug(inData, 'inData')
     const { oracleAddress, roundId, workerSource } = inData
 
+    if (!state.isActive({ oracleAddress })) {
+      logger.warn(`aggregatorJob for oracle ${oracleAddress} is no longer active. Removing job.`)
+      job.remove()
+      return 0
```

Comment:
Since the job does not return anything, I think it would be nicer to just use return without value.

```suggestion
      return
```